src/WebUI/Components/DeploymentsComponent.tsx

```diff
@@ -211,8 +232,27 @@ export class DeploymentsComponent extends BaseComponent<IDeploymentsComponentPro
                         <span className="deployment-status">{deployment.pods}</span>
                     </div>
                 );
+            case imageKey:
+                textToRender = deployment.image;
+                break;
+            case ageKey:
+            return (
+                <Ago date={new Date(deployment.creationTimeStamp)} />
+            );
         }
 
         return ListComponent.renderColumn(textToRender || "", ListComponent.defaultColumnRenderer, colDataClassName);
     }
+
+    private static _getHeadingContent( deployment:V1Deployment ): JSX.Element {
+        return (
+            <div>
+                <h3>{deployment.metadata.name}</h3>
+                <LabelGroup labelProps={Utils.getUILabelModelArray(deployment.metadata.labels)}

```

Comment:
<LabelGroup labelProps={Utils.getUILabelModelArray(deployment.metadata.labels)}
                    wrappingBehavior={WrappingBehavior.OneLine}
                    fadeOutOverflow={true}>
                </LabelGroup>

low prio: we should make this as a separate component, jsut a thought